.circleci/config.yml

```diff
@@ -188,6 +188,34 @@ jobs:
             git add --all
             git commit -m "Update documentation of commit ${CIRCLE_SHA1} [skip ci]" || true
             git push origin $BRANCH
+  docker-images:
+    docker:
+        - image: cimg/base:stable
+    resource_class: small
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker images
+          command: |
+            if git diff --name-only HEAD~1 HEAD | grep -q .circleci/images/bionic-setuptools; then
+                docker build -t integreat/bionic-setuptools:$(echo $CIRCLE_SHA1 | cut -c -7) -t integreat/bionic-setuptools:latest .circleci/images/bionic-setuptools
+            fi
+            if git diff --name-only HEAD~1 HEAD | grep -q .circleci/images/python-node-gettext; then
+                docker build -t integreat/python-node-gettext:$(echo $CIRCLE_SHA1 | cut -c -7) -t integreat/python-node-gettext:latest .circleci/images/python-node-gettext
+            fi
+      - run:
+          name: Push Docker images
+          command: |
+            if git diff --name-only HEAD~1 HEAD | grep -q .circleci/images; then
+                echo $DOCKER_PASSWORD | docker login -u integreat --password-stdin
```

Comment:
Does the `$DOCKER_PASSWORD` variable magically exist in Circle CI environments?

Reply:
The password is injected via a [CircleCI Context](https://app.circleci.com/settings/organization/github/Integreat/contexts).
I will explain this in detail when fixing #427 (Add CircleCI documentation).